path_oidc.go

```diff
@@ -161,6 +165,12 @@ func (b *jwtAuthBackend) pathCallback(ctx context.Context, req *logical.Request,
 		logFunc("error reading /userinfo endpoint", "error", err)
 	}
 
+	if role.VerboseOIDCLogging {
+		if c, err := json.Marshal(allClaims); err == nil {
```

Comment:
Would this ever fail to marshal? If it could, would it be useful to log the raw claims because of their unexpected state? Part of me thinks this would be super unlikely to fail, and would just return empty bytes anyways, so maybe in this situation it would be ok to just drop the err and log the potentially empty bytes.

Reply:
I'd put this at a near 0% chance, since `allClaims` is the result of json.Unmarshalling just a few lines earlier. But I will add a change to just dump the error to the log if by change there is one.